.github/CODEOWNERS

```diff
@@ -18,14 +18,18 @@
 
 ## CODEOWNERS for core/02-client
 
-/modules/core/02-client @seantking @damiannolan
-/proto/ibc/core/client @seantking @damiannolan
+/modules/core/02-client/ @seantking @damiannolan
+/proto/ibc/core/client/ @seantking @damiannolan
 
 # CODEOWNERS for the light-clients
 
 /modules/light-clients/ @colin-axner @fedekunze @AdityaSripal @seantking @damiannolan
 /proto/ibc/lightclients/ @colin-axner @fedekunze @AdityaSripal @seantking @damiannolan
 
+# CODEOWNERS for docs
+
+/docs/ @colin-axner @AdityaSripal @seantking @damiannolan @charleenfei @chatton @crodriguezvega
```

Comment:
Is this not included from the `*` at the top of file?

Reply:
True that! I guess then we don't need this PR! :D